english/explore-outside/journey-to-flaming-tower/journey-to-flaming-tower.md

```diff
@@ -18,4 +18,6 @@ Do you:
 
 [Take your cellphone out, call Tank and ask for the nearest exit out of the Matrix.](exit/exit.md)
 
-[Think about how your towel could have been useful.](towel/towel.md)
\ No newline at end of file
+[Think about how your towel could have been useful.](towel/towel.md)
+
+[Shout for help](shout/shout.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.